packages/ui/src/RadioFormik.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactElement, useContext } from 'react'
+import { FieldValidator, useField } from 'formik'
+
+import { Radio, RadioExtraProps } from './Radio'
+import { RadioGroupContext } from './RadioGroupContext'
+
+export type RadioFormikProps = RadioExtraProps & {
+  validate?: FieldValidator
```

Comment:
Why do we need a field validator here? We already have one at the `RadioGroupFormik` level. Even if this field validator errors the error would not be handled and displayed by us, would it?